Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN set -x \
     && echo "deb     https://sensu.global.ssl.fastly.net/apt stretch main" > /etc/apt/sources.list.d/sensu.list \
     && apt-get update \
     && apt-get install -y sensu=${SENSU_VERSION} \
-    && gem install --no-document \
+    && gem install --no-ri --no-rdoc \
```

Comment:
--no-document is the correct way to do this.  Your options are deprecated.

```
  Deprecated Options:
        --[no-]rdoc                  Generate RDoc for installed gems
                                     Use --document instead
        --[no-]ri                    Generate ri data for installed gems.
                                     Use --document instead
```

```
gem --version
2.0.14.1
```

Reply:
Ah, forgot about that.